activerecord/test/cases/adapters/abstract_mysql_adapter/connection_test.rb

```diff
@@ -212,8 +211,7 @@ def test_get_and_release_advisory_lock
   def test_release_non_existent_advisory_lock
     lock_name = "fake lock'n'name"
     released_non_existent_lock = @connection.release_advisory_lock(lock_name)
-    assert_equal released_non_existent_lock, false,
-      "expected release_advisory_lock to return false when there was no lock to release"
+    assert_equal(false, released_non_existent_lock, "expected release_advisory_lock to return false when there was no lock to release")
```

Comment:
Here too, let's keep the error message on a separate line like before:

```suggestion
    assert_equal false, released_non_existent_lock,
      "expected release_advisory_lock to return false when there was no lock to release"
```